tests/test_pair_potential.py

```diff
@@ -43,21 +43,21 @@ def test_pair_potential(generate_reference: bool) -> None:
 
     # Compare to reference data
     kets = [repr(ket) for ket in basis_pair.kets]
-    eigenvalues = np.array([system.get_eigenvalues(unit="GHz") for system in system_pairs])
-    eigenstates = np.array([system.get_eigenbasis().coefficients.todense().A1 for system in system_pairs])
+    eigenenergies = np.array([system.get_eigenenergies(unit="GHz") for system in system_pairs])
+    eigenbasis = np.array([system.get_eigenbasis().get_coefficients().todense().A1 for system in system_pairs])
```

Comment:
Here, I would stick with eigenstates because basis is for me an object like BasisAtom. Also at other places in this PR.